brokerlib.php

```diff
@@ -647,8 +647,12 @@ function bigbluebuttonbn_broker_recording_import($bbbsession, $params) {
 function bigbluebuttonbn_broker_meeting_events($bigbluebuttonbn) {
     // Decodes the received JWT string.
     try {
-        // Get the HTTP headers (getallheaders is a PHP function that may only work with Apache).
-        $headers = getallheaders();
+        // Get the HTTP headers - Apache webserver.
+        if (function_exists('apache_request_headers')) {
+            $headers = apache_request_headers();
+        } else { // Nginx webserver.
+            $headers = $_SERVER;
```

Comment:
Does this work for apache too? I'm not sure the if is needed

Reply:
The headers data might not exist in $_SERVER for apache, or is in a different format. 

This was previously a call to `getallheaders()` which wasn't available for nginx. I've updated it to be similar to the referenced code below

Additional references:
https://github.com/catalyst/moodle-webservice_restful/blob/master/locallib.php#L91